chpl/chpl-service/src/main/java/gov/healthit/chpl/user/cognito/password/CognitoForgotPassword.java

```diff
@@ -0,0 +1,51 @@
+package gov.healthit.chpl.user.cognito.password;
+
+import java.util.Date;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class CognitoForgotPassword {
+    private static final Long MILLIS_IN_HOUR = 24L * 60L * 60L * 1000L;
```

Comment:
The variable says milliseconds in an hour, but I think the numbers equal milliseconds in a day? Not sure which is correct.